scripts/scil_fibertube_score_tractogram.py

```diff
@@ -0,0 +1,213 @@
+reconstruction.
+
+VC: "Valid Connection": Represents a streamline that ended in the final
+    segment of the fibertube in which it was seeded.
+IC: "Invalid Connection": Represents a streamline that ended in the final
+    segment of another fibertube.
+NC: "No Connection": Contains streamlines that have not ended in the final
+    segment of any fibertube.
+
+A "coordinate absolute error" is the distance between a streamline coordinate
+and the closest point on its corresponding fibertube. The average of all
+coordinate absolute errors of a streamline is called the "Mean absolute
+error" or "mae".
+
+Computed metrics:
+    - truth_vc_ratio
+        Proportion of VC at ground-truth resolution.
+    - truth_ic_ratio
+        Proportion of IC at ground-truth resolution.
+    - truth_nc_ratio
+        Proportion of NC at ground-truth resolution.
+    - res_vc_ratio
+        Proportion of VC at the tracking sphere resolution.
+    - res_ic_ratio
+        Proportion of IC at the tracking sphere resolution.
+    - res_nc_ratio
+        Proportion of NC at the tracking sphere resolution.
+    - mae_min
+        Minimum MAE for the tractogram.
+    - mae_max
+        Maximum MAE for the tractogram.
+    - mae_mean
+        Average MAE for the tractogram.
+    - mae_med
+        Median MAE for the tractogram.
+
+See also:
+    - scil_ft_tracking.py to perform a fibertube tracking
+    - scil_tractogram_filter_collisions.py to prepare data for fibertube
+      tracking
+"""
+
+import os
+import json
+import argparse
+import logging
+import numpy as np
+import nibabel as nib
+
+from dipy.io.stateful_tractogram import StatefulTractogram, Space, Origin
+from dipy.io.streamline import save_tractogram, load_tractogram
+from scilpy.tractanalysis.fibertube_scoring import \
+    resolve_origin_seeding, endpoint_connectivity, mean_reconstruction_error
+from scilpy.tractograms.streamline_operations import \
+    get_streamlines_as_fixed_array
+from scilpy.io.utils import (assert_inputs_exist,
+                             assert_outputs_exist,
+                             add_overwrite_arg,
+                             add_verbose_arg,
+                             add_json_args)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('in_fibertubes',
+                   help='Path to the tractogram file containing the \n'
+                   'fibertubes with their respective diameter saved \n'
+                   'data_per_streamline (must be .trk). \n'
+                   'The fibertubes must be void of any collision \n'
+                   '(see scil_filter_intersections.py). \n')
+
+    p.add_argument('in_tractogram',
+                   help='Path to a text file containing the ground-truth \n'
+                   'reconstruction (must be .trk) with seeds saved as \n'
+                   'data_per_streamline.')
```

Comment:
I find very confusing the in_fibertubes vs in_tractogram options here and their description. 
in_fibertubes is the ground-truth, right? not the in_tractogram... 
in_tractogram comes from the scil_fibertube_tracking, no? 

Please clarify and make better description.

Reply:
I think it's better now, changed it for "in_tracking".